snuba/snuba_migrations/spans/0007_spans_add_profile_id.py

```diff
@@ -0,0 +1,57 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import UUID, Column
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set_name = StorageSetKey.SPANS
+local_table_name = "spans_local"
+dist_table_name = "spans_dist"
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Add profile_id column.
+    """
+
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=storage_set_name,
+                table_name=local_table_name,
+                column=Column(
+                    "profile_id", UUID(Modifiers(nullable=True, codecs=["ZSTD(1)"]))
```

Comment:
Not sure how much zstd compression would benefit on a UUID column. Not a blocker for the PR. Was just curious to know.

Reply:
Yeah there isn't much benefit. In the docs they mentioned using ZSTD over LZ4 by default is better.